fuzzers/007-timing/bel/pin_alias_map.json

```diff
@@ -0,0 +1,60 @@
+{
+    "ff_init": {
+        "Q": {
+             "names" : ["QL", "QH"],
+             "property" : true
+         }
+    },
+    "reg_init": {
+        "Q": {
+             "names" : ["QL", "QH"],
+             "property" : true
+         }
+    },
+    "rambfifo36e1" : {
+         "RSTRAMARSTRAMLRST" : {
+             "names" : ["rstramarstl"],
+             "property" : false
```

Comment:
"property"?  Not sure I understand what this field is for or how to use it?

Reply:
probably not the best name.

There are two types of aliases we need to handle. One is a pin name combined with a property e.g. `ql` or `qh` in `dff`. Here the actual pin name is `q` while `l` and `h` determines the `S` or `R` configuration of the `dff`. In this case we should emit a different celltype timing for the considered timing path (e.g. `DFF_S` or `DFF_R`)

The second type is when the actual BEL pin name is different than the one included in the timing string e.g. BRAM's `rstramau` from timing string refers to the `RSTRAMARSTRAMU` BEL pin. In this case we simply translate the pin name.

Maybe this field should be renamed to something like `is_property_related`?